quickstart.md

```diff
@@ -161,253 +162,55 @@ NAME            HANDLER         AGE
-
-This is not the case for most real workloads. It is recommended to use CoCo with signed and/or encrypted images.
-The workload itself can also request secrets from the attestation agent in the guest.
-
-Secrets are provisioned to the guest in conjunction with an attestation, which is based on hardware evidence.
-The rest of this guide focuses on setting up more substantial encrypted/signed workloads using attestation
-and confidential hardware.
+certain platforms may require additional configuration to enable confidential computing. For example, a specific 
+host kernel or firmware may be required. See the [guides](./guides/) for more information.
 
-CoCo has a modular attestation interface and there are a few options for attestation.
-CoCo provides a generic Key Broker Service (KBS) that the rest of this guide will be focused on.
-The SEV runtime class uses `simple-kbs`, which is described in the [SEV guide](./guides/sev.md).
-There is also `eaa_kbc`/`verdictd` which is described [here](./guides/eaa_verdictd.md).
+## Using CoCo
 
-### Select Runtime Class
+Below is a brief summary and description of some of the CoCo use cases:
 
-To use CoCo with confidential hardware, first switch to the appropriate runtime class.
-TDX has two runtime classes, `kata-qemu-tdx` and `kata-clh-tdx`. One uses QEMU as VMM and TDVF as firmware. The other uses Cloud Hypervisor as VMM and TD-Shim as firmware.
+- **Container Launch with Only Memory Encryption (No Attestation)** - Launch a container with memory encryption 
+- **Container Launch with Encrypted Image** - Launch an encrypted container by proving the workload is running 
+  in a TEE in order to retrieve the decryption key
+- **Container Launch with Image Signature Verification** - Launch a container and verify the authenticity and 
+  integrity of an image by proving the workload is running in a TEE
+- **Sealed secret** - Implement wrapped kubernetes secrets that are confidential to the workload owner and are
+  automatically decrypted by proving the workload is running in a TEE
 
-For SEV(-ES) use the `kata-qemu-sev` runtime class and follow the [SEV guide](./guides/sev.md).
+The CoCo use cases are implemented differently for each TEE, and they are described in each corresponding 
+[guide](./guides) section. To get started using CoCo without TEE hardware, follow the Non-TEE guide below:
 
-For `enclave-cc` follow the [enclave-cc guide](./guides/enclave-cc.md).
+- [CoCo-dev](./guides/coco-dev.md)
+- [SEV(-ES)](./guides/sev.md)
+- [SNP](./guides/snp.md)
+- TDX
+- [SGX](./guides/enclave-cc.md)
+- Secure Enclave
+- [IBM Secure Execution](./guides/ibm-se.md)
+- ...
 
-### Deploy and Configure tenant-side CoCo Key Broker System cluster
+## Additonal CoCo Features
 
-The following describes how to run and provision the generic KBS.
-The KBS should be run in a trusted environment. The KBS is not just one service,
-but a combination of several.
+Here are some of the additional features CoCo supports:
```

Comment:
We were trying to distinguish major orchestration use cases / flows from supplemental or complimentary features. If you think there's a better way to represent this, we can do that. Or just combine these with the use cases. What you think @stevenhorsman?

Reply:
I'm just not sure I understand why some of them are major use cases/flows and others are supplemental? Where did that categorisation come from e.g. sealed secrets is a features and secure storage is a feature, what's the difference?

Reply:
Like I mentioned, was trying to distinguish orchestration use case (k8s sealed secret) from a feature that may be complimentary but not required like secure storage. Some of this may be coming from our limited understanding of these additional items. We're just trying to provide a framework to make things more clear in the documentation. Steve, you're prob more familiar with all the features, what makes sense here?